tpm/tpm.go

```diff
@@ -61,6 +61,21 @@ func OpenTPM(path string) (io.ReadWriteCloser, error) {
 	return rwc, nil
 }
 
+// GetKeys gets the list of handles for currently-loaded TPM keys.
+func GetKeys(f *os.File) ([]Handle, error) {
```

Comment:
Please also add an example/test in tpm_test.go for GetKeys. It will Skip() if openTPMOrSkip can't reach the TPM.